includes/Story_Post_Type.php

```diff
@@ -174,9 +181,10 @@ public function register_post_type() {
 				'rewrite'               => [
 					'slug'       => self::REWRITE_SLUG,
```

Comment:
```suggestion
					'slug'       => is_string( $this->get_has_archive() ) ? $this->get_has_archive() : self::REWRITE_SLUG,
```

I think we need to this, so the url structure of stories changed as well.

Reply:
This slug is completely ignored for archives if `has_archive` is a string. See https://github.com/WordPress/wordpress-develop/blob/0f0089391b0f92d4508b8a4df567df2052161340/src/wp-includes/class-wp-post-type.php#L605

Changing the URL structure for individual stories is out of scope.